packages/docs/src/routes/docs/integrations/astro/index.mdx

```diff
@@ -0,0 +1,238 @@
+---
+title: Astro | Integrations
+contributors:
+  - thejackshelton
+---
+
+# Astro
+
+Astro is a flexible framework that accommodates a wide range of tools and integrations, enabling you to leverage numerous ecosystems.
+
+It also allows you to write components using your favorite UI framework (or no framework at all) and renders your pages to static HTML at build time, or dynamically on the server using Server-Side Rendering (SSR). 
+
+This results in a fast, SEO-friendly output that can be deployed to any static hosting environment or server.
+
+## Astro instead of Qwik City
+
+When integrating Qwik with Astro, it's important to note that Qwik City APIs are not compatible with Astro. 
```

Comment:
Maybe we should switch it around?
`when integrating Astro with Qwik`
also, I think we should hi-light this more, maybe put it in bold?